### ISSUE TITLE
Build failed: Cannot read property 'map' of undefined

### ISSUE THREAD

**Issue by guivazcabral** [argos-ci/argos-javascript#34]
I am running cypress to generate screenshots with `cy.argosScreenshot("file name");` on a react project (running on node 14) and when I try to run `argos upload cypress/screenshots` it fails with an error:

```
✖ Build failed: Cannot read property 'map' of undefined
```

can you help please? thanks

**Comment by jsfez** [comment]
Hello @guivazcabral,
It's not easy to help you with this much information.

Do you have a link to check the repository ? 

If it's a private repository contact me on [Argos Discord server](https://discord.gg/M8ATmvqv). I'll try to help you.

**Comment by guivazcabral** [comment]
Hi, @jsfez. Thank you for your reply!

I tried setting up an example repo and replicate the issue but I wasn't able to do it. I figured it had to be a problem on my project. After doing some digging, adding some logs to the argos cli binary, I found things were breaking when loading the screenshots with fast-glob. Indeed, I have other dependencies using different versions of fast-glob. My guess (and just a guess) is that for some reason the wrong version of fast glob was being loaded.

The problem was solved by installing fast-glob on the project itself with the same version that argos cli uses.
So uploading is now working. Thanks for your time :D
